From d3e153ad404b3f23c17cd0510b08a149207e5cd1 Mon Sep 17 00:00:00 2001
From: Jean-Michel Trivi <jmtrivi@google.com>
Date: Thu, 7 Nov 2024 15:52:39 -0800
Subject: [PATCH 229/436] Add flag for spatial audio versioning

Flag: android.media.audio.spatial_audio_settings_versioning
Test: compile
Bug: 377977731
Change-Id: I3495701471109d773fa9a40f82ad3e0fa8f8cfc9
---
 media/audio/aconfig/audio_framework.aconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/media/audio/aconfig/audio_framework.aconfig b/media/audio/aconfig/audio_framework.aconfig
index 7ee1360501..ae7eb36644 100644
--- a/media/audio/aconfig/audio_framework.aconfig
+++ b/media/audio/aconfig/audio_framework.aconfig
@@ -198,6 +198,13 @@ flag {
     bug: "367816690"
 }
 
+flag {
+    name: "spatial_audio_settings_versioning"
+    namespace: "media_audio"
+    description: "introduce versioning of spatial audio settings"
+    bug: "377977731"
+}
+
 flag {
     name: "spatializer_capabilities"
     namespace: "media_audio"
-- 
2.43.0

